public/js/src/module/gui.js

```diff
@@ -56,11 +56,7 @@ function init() {
 function setEventHandlers() {
     const $doc = $(document);
 
-    $doc.on('click', '#feedback-bar .close, .touch #feedback-bar', () => {
-        feedbackBar.hide();
-
-        return false;
-    });
+    feedbackBar.setCloseHandler();
```

Comment:
I considered letting the feedback-bar module deal with adding the listener e.g. when the `show` method is first called. Do you think that would be better? I left it here to stick with "how things are done" in gui.js.